spec/models/season/phase_switcher_spec.rb

```diff
@@ -26,9 +25,11 @@
   end
 
   describe '#fake_application_phase' do
+
     it 'timeshifts the application phase to today' do
+      phase = :fake_application_phase
       Timecop.travel(Season.current.applications_close_at - 1.month) do
-        Season::PhaseSwitcher.fake_application_phase
+        Season::PhaseSwitcher.destined(phase)
         expect(subject).to be_application_period
       end
```

Comment:
I thought we want to test the results of the incoming call (from `.destined`, instead of  :fake_*_phase). Is that right?